### PR TITLE
Original Links into Datasets

### PR DESCRIPTION
**Pull request by patfire12345**
https://github.com/TBS-EACPD/infobase/issues/1172

### CODE REVIEW

**Review by ktw1016 — COMMENTED** [review]
Screen Shot 2021-07-07 at 3 20 38 PM

I think just matching how the button style look better, what do you think?

File: client/src/metadata/data_sources.js [hunk 1]
```diff
@@ -80,6 +80,10 @@ const sources = _.chain([
     title: text_maker("pa_title"),
     frequency: text_maker("yearly"),
     open_data: infobase_open_data_page,
+    original: {
```
[the remark this answers is not in the document]
Comment (patfire12345) [reply client/src/metadata/data_sources.js]: Done!

**Comment by patfire12345** [comment]
> Screen Shot 2021-07-07 at 3 20 38 PM
> 
> I think just matching how the button style look better, what do you think?

Yeah, I think so too, I did it like that because Ross' original issue was similar to the previous button, but I've changed it to that now!

**Review by ktw1016 — APPROVED** [review]
Looks good to me, I'd just ask Ross to check over the dev link before merging in

**Comment by patfire12345** [comment]
Closes #1172

**Pull request merged**